explorer/client/src/components/displaybox/DisplayBox.module.css

```diff
@@ -4,10 +4,61 @@ div.imagebox {
 
 img.imagebox {
     @apply object-scale-down w-[100%];
+
+    filter: blur(0);
+    opacity: 1;
+    transition: opacity ease-out 0.14s;
+    transition: filter ease-in 0.14s;
+}
+
+/* ambient filter pattern for after image is loaded but hasn't been auto-modded yet */
+@keyframes changeblur {
+    from {
+        filter: blur(15px) contrast(90%);
+        opacity: 0.72;
+    }
+
+    to {
+        filter: blur(10px) contrast(60%);
+        opacity: 0.64;
+    }
+}
+
+img.imageboxblur {
```

Comment:
it could take longer sometimes: the loop seems fine to me.  we could also have a loading message, this was just to imply that something is still happening behind the scenes